UM/Settings/ContainerRegistry.py

```diff
@@ -166,36 +167,58 @@ def addContainerType(cls, container):
     #   If this function is called again, it will clear the old data and reload.
```

Comment:
It is not even correct anymore because we actually do not clear the existing list of containers.